### PR TITLE
openstack: Add a 4.9 OVN daily periodic

### PR DESCRIPTION
**Pull request by pierreprinetti**
Implements [OSASINFRA-2535](https://issues.redhat.com/browse/OSASINFRA-2535)

### CODE REVIEW

**Comment by pierreprinetti** [comment]
I think it's important to have this test, _especially_ since it's currently failing. WDYT @mandre

**Comment by pierreprinetti** [comment]
@EmilienM Please remove the /hold when you feel it's time to merge.

**Review by mandre — COMMENTED** [review]
/lgtm
/hold cancel

**Comment by mandre** [comment]
You may want to add this job to the test grid as well.

**Comment by pierreprinetti** [comment]
/assign @ravisantoshgudimetla
like the bot suggests

File: ci-operator/config/openshift/release/openshift-release-master__ci-4.9.yaml [hunk 1]
```diff
@@ -208,6 +208,14 @@ tests:
     env:
       TEST_SUITE: openshift/conformance/serial
     workflow: openshift-e2e-openstack-ipi
+- as: e2e-openstack-ovn
+  cron: '@daily'
```
Comment (dmage) [inline ci-operator/config/openshift/release/openshift-release-master__ci-4.9.yaml:212]: `interval: 24h`

Comment (dmage) [reply ci-operator/config/openshift/release/openshift-release-master__ci-4.9.yaml:212]: Some our tools don't understand `@daily` and won't be able to estimate how many historical records should be kept. Therefore `interval` is better than `cron`.

Comment (EmilienM) [reply ci-operator/config/openshift/release/openshift-release-master__ci-4.9.yaml]: @dmage done! thanks for the great suggestion :)

**Comment by pierreprinetti** [comment]
Thanks!
I wonder if I can LGTM now that I am no longer committer?

/lgtm

**Pull request merged**